src/components/shared/TreeView.css

```diff
@@ -173,7 +173,7 @@
   color: black;
 }
 
-.treeViewRow.dim > .treeViewMainColumn {
+.treeViewMainColumn.dim {
```

Comment:
I simplified this while I'm here, because this is the only element being dimmed. In the past we were dimming more elements so this made sense to have the class higher in the tree.